### PR TITLE
Add more to docs

### PR DESCRIPTION
**Pull request by Kanti**
#99

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +64,28 @@ $config->addAppInstance('local')
...
+## CLI Examples
+
+````bash
+phpsu sync production --dry-run
+phpsu sync p --no-db
+phpsu sync p --no-fs
+phpsu sync production testing --all
+````
+
+### PHP API Examples
```
Comment (ChrisB9) [inline README.md]: this we should leave out for now and put more though on it in the next milestone. Otherwise i'd approve it

Comment (Kanti) [inline README.md]:
```suggestion
<!--### PHP API Examples
```

File: README.md [hunk 2]
```diff
@@ -64,6 +64,28 @@ $config->addAppInstance('local')
...
+````php
+<?php
+declare(strict_types=1);
+
+$log = new \Symfony\Component\Console\Output\BufferedOutput();
+$configurationLoader = new \PHPSu\Config\ConfigurationLoader();
+$syncOptions = new \PHPSu\SyncOptions('production');
+$phpsu = new \PHPSu\Controller();
+$phpsu->sync($log, $configurationLoader->getConfig(), $syncOptions);
+````
```
Comment (Kanti) [inline README.md]:
```suggestion
````-->
```

File: tests/ControllerTest.php [hunk 3]
```diff
@@ -210,6 +211,22 @@ public function testUseCaseDatabaseOnlyDefinedOnOneEnd(): void
         $this->assertSame($lines, explode("\n", $output->fetch()));
     }
 
+    public function testPhpApiReadmeExample()
```
Comment (ChrisB9) [inline tests/ControllerTest.php]: since we commented this out in the readme, i dont think we should for now have this in the tests either

Comment (Kanti) [reply tests/ControllerTest.php]: A test more dosen't hurt.

**Pull request merged**